receiver/haproxyreceiver/README.md

```diff
@@ -1,10 +1,6 @@
 # HAProxy Receiver
 
-| Status                   |           |
-| ------------------------ |-----------|
-| Stability                | [alpha]   |
-| Supported pipeline types | metrics   |
-| Distributions            | [contrib] |
+Component [status](./documentation.md#haproxyreceiver).
```

Comment:
I like having the status being very much top of the README. Is there a way to apply the same approach taken by `table-generation` in the specification repository?

Reply:
I've updated the code to insert the status table into the readme. it uses a regex to find the following start/end lines in the markdown:

```
<!-- status autogenerated section -->
<!-- end autogenerated section -->
```